github/actions_runners.go

```diff
@@ -214,6 +214,50 @@ func (s *ActionsService) CreateOrganizationRegistrationToken(ctx context.Context
 	return registrationToken, resp, nil
 }
 
+// CreateEnterpriseRegistrationToken creates a token that can be used to add a self-hosted runner to an enterprise.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/enterprise-admin#create-a-registration-token-for-an-enterprise
+func (s *ActionsService) CreateEnterpriseRegistrationToken(ctx context.Context, enterprise string) (*RegistrationToken, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/actions/runners/registration-token", enterprise)
+
+	req, err := s.client.NewRequest("POST", u, nil)
```

Comment:
I'm confused.  It looks to me like we already support this endpoint:

https://github.com/google/go-github/blob/master/github/enterprise_actions_runners.go#L17